src/client/dfuse/dfuse_main.c

```diff
@@ -226,94 +248,98 @@ show_version(char *name)
 static void
 show_help(char *name)
 {
-	printf("usage: %s -m mountpoint\n"
-		"Options:\n"
-		"\n"
-		"	-m --mountpoint=<path>	Mount point to use\n"
-		"\n"
-		"	   --pool=name		pool UUID/label\n"
-		"	   --container=name	container UUID/label\n"
-		"	   --path=<path>	Path to load UNS pool/container data\n"
-		"	   --sys-name=STR	DAOS system name context for servers\n"
-		"\n"
-		"	-S --singlethread	Single threaded\n"
-		"	-t --thread-count=count	Number of fuse threads to use\n"
-		"	-f --foreground		Run in foreground\n"
-		"	   --enable-caching	Enable all caching (default)\n"
-		"	   --enable-wb-cache	Use write-back cache rather than write-through (default)\n"
-		"	   --disable-caching	Disable all caching\n"
-		"	   --disable-wb-cache	Use write-through rather than write-back cache\n"
-		"\n"
-		"	-h --help		Show this help\n"
-		"	-v --version		Show version\n"
-		"\n"
-		"Specifying pool and container are optional. If not set then dfuse can connect to\n"
-		"many using the uuids as leading components of the path.\n"
-		"Pools and containers can be specified using either uuids or labels.\n"
-		"\n"
-		"The path option can be use to set a filesystem path from which Namespace attributes\n"
-		"will be loaded, or if path is not set then the mount directory will also be\n"
-		"checked.  Only one way of setting pool and container data should be used.\n"
-		"\n"
-		"The default thread count is one per available core to allow maximum throughput,\n"
-		"this can be modified by running dfuse in a cpuset via numactl or similar tools.\n"
-		"One thread will be started for asynchronous I/O handling so at least two threads\n"
-		"must be specified in all cases.\n"
-		"Singlethreaded mode will use the libfuse loop to handle requests rather than the\n"
-		"threading logic in dfuse."
-		"\n"
-		"If dfuse is running in background mode (the default unless launched via mpirun)\n"
-		"then it will stay in the foreground until the mount is registered with the\n"
-		"kernel to allow appropriate error reporting.\n"
-		"\n"
-		"Caching is on by default with short metadata timeouts and write-back data cache,\n"
-		"this can be disabled entirely for the mount by the use of command line options.\n"
-		"Further settings can be set on a per-container basis via the use of container\n"
-		"attributes.  If the --disable-caching option is given then no caching will be\n"
-		"performed and the container attributes are not used, if --disable-wb-cache is\n"
-		"given the data caching for the whole mount is performed in write-back mode and\n"
-		"the container attributes are still used\n"
-		"\n"
-		"version: %s\n",
-		name, DAOS_VERSION);
+	printf(
+	    "usage: %s -m mountpoint\n"
```

Comment:
Should we show positional arg usage here to encourage it?
```suggestion
	    "usage: %s <mountpoint> [pool] [cont]\n"
```